README.md

```diff
@@ -1,93 +1,67 @@
 # `tag-release`
 
+## Installation
+```
+$ npm install -g tag-release
+```
+
 ## Usage
 
-### No Flags
+### Default
+```
+$ tag-release
+```
 
-You will be prompted for information regarding what type of update you are
+You will be prompted for information regarding what type of release you are
 making (major, minor, patch). Along the way, you'll have the opportunity to
 verify, modify, or cancel the `tag-release` process.
 
+### Commands
 ```
-$ tag-release
-
-? What is your GitHub username
-? What is your GitHub password
-? What type of release is this (Use arrow keys)
-❯ Major (Breaking Change)
-  Minor (New Feature)
-  Patch (Bug Fix)
-etc...
+$ tag-release --help
 ```
 
-### With Flags
+Will display a list of avaliable commands and options.
 
 ```
-$ tag-release --help
-$ tag-release -h
-
-Usage: tag-release [options]
-
-Options:
-
-  -h, --help                     Output usage information
-  -c, --config [filePath]        Path to JSON Configuration file (defaults to './package.json')
-  -r, --release [type]           Release type (major, minor, patch)
-  --verbose                      Console additional information
-  -V, --version                  Output the version number
-  -p, --prerelease               Create a pre-release
-  -i, --identifier <identifier>  Identifier used for pre-release
-  --reset                        Reset repo to upstream master/develop branches
-  --promote [tag]                Promotes specified pre-release tag to an offical release
-  --continue                     Continues the rebase process of a tag promotion
-  --qa [scope]                   Create initial upstream branch for upstream repo
-  --pr [scope]                   Update package.json to bump upstream repos and create a PR to develop
-  --maxbuffer <n>                Overrides the max stdout buffer of the child process. Size is 1024 * <n>.
-
-Examples:
-
-   $ tag-release
-   $ tag-release --config ../../config.json
-   $ tag-release -c ./manifest.json
-   $ tag-release --release major
-   $ tag-release -r minor
-   $ tag-release --verbose
-   $ tag-release -V
-   $ tag-release --version
-   $ tag-release --prerelease
-   $ tag-release -p
-   $ tag-release -p -i foo
-   $ tag-release --reset
-   $ tag-release --promote
-   $ tag-release --promote v1.1.1-my-tagged-version.0
-   $ tag-release --continue
-   $ tag-release --qa
-   $ tag-release --qa myorg
-   $ tag-release --pr
-   $ tag-release --pr myorg
-   $ tag-release --maxbuffer 5000
+  Usage: tag-release [options] [command]
+
+
+  Options:
+
+    -V, --version  output the version number
+    -h, --help     output usage information
```

Comment:
hmm, does it not show `--maxbuffer` as an option? or does that only show for help on one specific command?

Reply:
If you want to show that option, we would to do `--help` off of he command that it exists off of.